source/help/apps/desktop-guide.rst

```diff
@@ -4,32 +4,32 @@ Using Mattermost Desktop App
 
 Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
-Below are a few tips to get you started and configuring your experience on the desktop app:
+Below are a few tips to get you started and to configure your experience on the desktop app:
 
  - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
  - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
  - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
- - `Troubleshooting <https://docs.mattermost.com/help/apps/desktop-guide.html#id4>`_
 
 Team Management
 ---------------------------------------------------------------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-Team Management appears in the Settings page, available from the `File` menu under `Settings`.
+The Team Management section allows you to add, edit and remove servers and teams.
 
 Adding servers and teams
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To add a new server on the Mattermost instance, 
 
-1. Click **Add new team** next to the right of Team Management section.
-2. Enter **Name** and a valid **URL**, which begins with either ``http://`` or ``https://``.
-3. Click **Add**.
+1. Go to **File** > **Settings** or press CTRL/CMD+COMMA
+2. Click **Add new team** next to the right of Team Management section.
+3. Enter **Name** and a valid **URL**, which begins with either ``http://`` or ``https://``.
+4. Click **Add**.
 
 You can now access all teams you have joined in the server once you have saved the settings, and each will appear as a separate tab in the app. 
 
```

Comment:
That's correct -- see comment in https://github.com/mattermost/docs/pull/790#discussion_r97192597